Makefile

```
@@ -2,6 +2,16 @@
 Q=$(if $V,,@)
 SRC=$(shell find . -type f -name '*.go')
 
+# protoc-gen-go constraints
+GEN_GO_BIN ?= protoc-gen-go
+GEN_GO_MIN_VERSION ?= 1.31.0
+GEN_GO_VERSION ?= $(shell $(GEN_GO_BIN) --version | awk --field-separator=' v' '{print $$NF}')
```

This didn't work for me on macOS nor in Ubuntu 23.04, but this works:

```suggestion
GEN_GO_VERSION ?= $(shell $(GEN_GO_BIN) --version | awk -F ' v' '{print $$NF}')
```

---

weird, I'm on ubuntu myself and the action's also running on ubuntu in CI.